src/how_to/nominate.md

```diff
@@ -1,15 +1,39 @@
 # Nominate an issue
 
-- Nomination is currently supported on the following repositories:
-  - rust-lang/rfcs
-  - rust-lang/rust
-  - rust-lang/reference
-  - rust-lang/lang-team
-  - (this set is defined by the `nominated` list in the [triagebot source code](https://github.com/rust-lang/triagebot/blob/master/src/agenda.rs))
-- Tag the issue with the `I-nominated` and `T-lang` labels by issuing the following [rustbot] command:
-  - `@rustbot label +I-nominate +T-lang`
-- Add a comment explaining
-  - What question you would like answered by the lang team
-  - Relevant background info or links to relevant info that the lang team can use to get "oriented" in the discussion
+You can raise issues to lang team attention by tagging them with `I-lang-nominated`. We scan through nominated issues during our [triage meetings](../meetings/triage.md). For each issue, we try to answer questions and reach decisions on the question being discussed in the issue.
+
+## How to nominate
+
+Add a **self-sufficient** comment to the issue that explains why you are nominated the issue and includes the text `@rustbot label +I-lang-nominated`. For example:
+
+```
+@rustbot label +I-lang-nominated
+
+I am nominating this for lang-team attention. We have been discussing the pros/cons of updating the type-checking rules for foo bar. The options on the table are as follows:
+
+* Allow type mismatches: This is good because blah.
+* Disallow type mismatches: This is good because blah.
+
+Where should we go from here?
+```
+
+The ideal comment will identify precisely what question you would like answered. Please try to make the comment easy for us to parse and understand without requiring a lot of context. We encourage links to internals or Zulip so we can dive into the details, but it really helps us give useful answers if you can summarize the key details up front.
```

Comment:
```suggestion
The ideal comment will identify precisely what question you would like answered. Please try to make the comment easy for us to parse and understand without requiring a lot of context. We encourage links to internals or Zulip so we can dive into the details, but it really helps us give useful answers if you can summarize the key details up front.

If your comment requires more than 5-10 minutes of reading and discussion to understand and effectively respond to, consider filing a meeting proposal (TODO: link) instead. This will give you ~60 minutes to present your question and the lang team more time to analyze it. We may punt the question back to you with an ask to do so if the question isn't answerable in our triage meeting time.
```